sweep/sweeper.go

```diff
@@ -799,54 +799,189 @@ func (s *UtxoSweeper) sweep(set InputSet) error {
 		s.currentOutputScript = pkScript
 	}
 
-	// Create sweep tx.
-	tx, fee, err := createSweepTx(
-		set, nil, s.currentOutputScript, uint32(s.currentHeight),
-		s.cfg.MaxFeeRate.FeePerKWeight(), s.cfg.Signer,
-	)
-	if err != nil {
-		return fmt.Errorf("create sweep tx: %v", err)
+	// Create a fee bump request and ask the publisher to broadcast it. The
+	// publisher will then take over and start monitoring the tx for
+	// potential fee bump.
+	req := &BumpRequest{
```

Comment:
Feels like we're conflating fee bumping (shrinking outputs or adding more inputs to increase fees) with the entire process of sweeping (adding inputs, signing, etc, etc).